code/game/objects/items/sharpener.dm

```diff
@@ -1,57 +1,77 @@
+/**
+* # Whetstone
+*
+* Items used for sharpening stuff
+*
+* Whetstones can be used to increase an item's force, throw_force and wound_bonus and it's change it's sharpness to SHARP_EDGED. Whetstones do not work with energy weapons. Two-handed weapons will only get the throw_force bonus. A whetstone can only be used once.
+*
+*/
 /obj/item/sharpener
 	name = "whetstone"
 	icon = 'icons/obj/kitchen.dmi'
 	icon_state = "sharpener"
 	desc = "A block that makes things sharp."
-	force = 5
-	var/used = 0
+	force = 5 //This is how hard the whetstone itself hits stuff
+	///If FALSE, the whetstone can be used. If TRUE, the whetstone is a worn whetstone.
+	var/used = FALSE
+	///How much force the whetstone can add to an item.
 	var/increment = 4
+	///Maximum force sharpening items with the whetstone can result in
 	var/max = 30
+	///The prefix a whetstone applies when an item is sharpened with it
 	var/prefix = "sharpened"
-	var/requires_sharpness = 1
-
+	///If TRUE, the whetstone will only sharpen already sharp items
+	var/requires_sharpness = TRUE
 
 /obj/item/sharpener/attackby(obj/item/I, mob/user, params)
-	if(used)
+	if(used) //You can't sharpen stuff with a worn whetstone
 		to_chat(user, "<span class='warning'>The sharpening block is too worn to use again!</span>")
 		return
-	if(I.force >= max || I.throwforce >= max)//no esword sharpening
+	if(I.force >= max || I.throwforce >= max) //If we're trying to sharpen an item that already has a force higher than max, we fail
 		to_chat(user, "<span class='warning'>[I] is much too powerful to sharpen further!</span>")
 		return
-	if(requires_sharpness && !I.get_sharpness())
+	if(requires_sharpness && !I.get_sharpness()) //You can't sharpen an item that isn't sharp with a whetstone that requieres items to be sharp
 		to_chat(user, "<span class='warning'>You can only sharpen items that are already sharp, such as knives!</span>")
 		return
-	if(is_type_in_list(I, list(/obj/item/melee/transforming/energy, /obj/item/dualsaber)))
+	if(is_type_in_list(I, list(/obj/item/melee/transforming/energy, /obj/item/dualsaber))) //You can't sharpen photons
 		to_chat(user, "<span class='warning'>You don't think \the [I] will be the thing getting modified if you use it on \the [src]!</span>")
 		return
 
-	var/signal_out = SEND_SIGNAL(I, COMSIG_ITEM_SHARPEN_ACT, increment, max)
-	if(signal_out & COMPONENT_BLOCK_SHARPEN_MAXED)
+	var/signal_out = SEND_SIGNAL(I, COMSIG_ITEM_SHARPEN_ACT, increment, max) //This is used to check more things if the item has a relevant component. As of December 2020, this is only used for two_handed and transforming.
```

Comment:
In future things will change, and people will be forced to check anyway. Better to leave out stuff that can become dated easily. Might be better to describe what signal_out is, rather then the purpose of the whole block. that should become clear as people read it.